test/math.jl

```diff
@@ -4,6 +4,12 @@
 @test significand(12.8) == 1.6
 @test exponent(12.8) == 3
 
+@test_approx_eq deg2rad(180) 1pi
```

Comment:
since we have separate `deg2rad` methods for different types, we probably want to test them all.  You can do something like:
`for T in (Int, Float64, BigFloat)
      @test_approx_eq deg2rad(T(180)) 1pi
      @test_approx_eq deg2rad(T[45, 60]) [pi/T(4), pi/T(3)]
      ....
end
```
to loop the test over different types.